src/resources/views-elfinder/elfinder.blade.php

```diff
@@ -1,25 +1,25 @@
 @extends('backpack::layout')
 
 @section('after_scripts')
-    <!-- jQuery and jQuery UI (REQUIRED) -->
+    {{-- jQuery and jQuery UI (REQUIRED) --}}
     <link rel="stylesheet" href="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/themes/smoothness/jquery-ui.css" />
-    <!-- <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script> -->
+    {{-- <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script> --}}
```

Comment:
It's either one or the other - personally I'd just **delete** the relevant one here.